dnsmasq.conf.ipv6

```diff
@@ -0,0 +1,21 @@
+interface=PROVISIONING_INTERFACE
+bind-dynamic
+enable-tftp
+tftp-root=/shared/tftpboot
+log-dhcp
+
+enable-ra
```

Comment:
Note sure if this is needed, it might be whats causing PXE to sometimes work i.e. depends on what happens first dhcpv6 or slaac...